src/common.jl

```diff
@@ -0,0 +1,51 @@
+abstract DASSLDAEAlgorithm <: AbstractODEAlgorithm
+immutable dassl <: DASSLDAEAlgorithm
```

Comment:
Right, the name clash with the package is not easily fixable.  I think this PR is good to go.